nefertari/es_query.py

```diff
@@ -416,15 +435,17 @@ def __call__(self, item):
             WildCardProcessor(),
             MatchProcessor(),
             FinalProcessor(),
+            NestedFieldProcessor(),
```

Comment:
In this case order is not important, each processor know which processor it should call next.